src/name_core.jl

```diff
@@ -288,7 +281,8 @@ function unify_names_longest(names_a, names_b)
     # 0 Allocations: @btime (()-> unify_names_longest((:a,:b), (:a,)))()
 
     length(names_a) === length(names_b) && return unify_names(names_a, names_b)
-    long, short = length(names_a) > length(names_b) ? (names_a, names_b) : (names_b, names_a)
+    long, short =
+        length(names_a) > length(names_b) ? (names_a, names_b) : (names_b, names_a)
```

Comment:
I think it's that this should become
```julia
long, short = if length(names_a) > length(names_b)
    (names_a, names_b)
else
   (names_b, names_a)
end
```
and it's on the to-do list https://github.com/domluna/JuliaFormatter.jl/issues/283#issue-694066362

So it's fine that the formatter did this for now -- it's one of the not-yet-implemented things.